labelplus/gtkui/common/gtklib/dnd.py

```diff
@@ -655,14 +653,9 @@ def _find_target(self, widget, context):
           if src and not same:
 
             targets = src.drag_source_get_target_list() or Gtk.TargetList.new()
-            for t in targets:
-
-              if t[0] == target.name:
-
-                if t[1] & Gtk.TargetFlags.SAME_WIDGET:
-                  target = None
-
-                break
```

Comment:
Do I understand correctly this was all wrong to begin with, as `targets` is of `Gtk.TargetList` (https://lazka.github.io/pgi-docs/Gtk-3.0/classes/TargetList.html#Gtk.TargetList) type containing `TargetPair`s, meaning accessing fields via `t[index]` was nonsense?